inspirehep/modules/theme/views.py

```diff
@@ -646,6 +646,10 @@ def ajax_experiments_people():
         }
     )
 
+@blueprint.route('/account/settings/linkedaccounts/', methods=['GET'])
+def linkedaccounts():
+    """."""
```

Comment:
I remember telling to myself I would write something there, did not happen apparently :P